examples/odf/main.tf

```diff
@@ -1,17 +1,11 @@
 provider "ibm" {
-  region = var.region
 }
 
 // Module:
-
 module "odf" {
-  source = "./../../modules/odf"
-  // TODO: With Terraform 0.13 replace the parameter 'enable' or the conditional expression using 'with_iaf' with 'count'
-  enable       = var.enable
-  cluster_id   = var.cluster_id
-  roks_version = var.roks_version
-
-  // Cluster parameters
-  //kube_config_path = data.ibm_container_cluster_config.cluster_config.config_file_path
-  kube_config_path = var.kube_config_path
+  source = "./.."
```

Comment:
Is this correct?   Will this pick up the module code?  Shouldn't it still be `source = "./../../modules/odf"`

Reply:
done